### DISCUSSION TITLE
Ajuda : Erro  401 nas requisições post (url codespace ) exigindo autenticação.

### DISCUSSION THREAD

**Discussion by NayaraNicacio** [community/community#138565] [Programming Help]
### Body

Estou utilizando o codespace e o postman para realizar testes de requisição, porem quando realizo um post o postman pede autenticação, ja tentei criar chave publica mais sem sucesso ainda nas requisições. ( PS: Rodando o postman no navegador )

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Salgado2004** [comment]
Boa tarde @NayaraNicacio 

Eu não trabalhei muito usando codespaces... 🤔 mas nunca tive esse problema. Como você está tentando acessar a aplicação? Pelo localhost ou pela URL gerada pelo codespace?

Se for a URL, acredito que o erro de autorização pode estar relacionado... ai recomendo fazer um teste usando o Localhost para acessar a aplicação
> Tem um tutorial de como fazer isso em **[Forwarding-Ports](https://docs.github.com/en/codespaces/developing-in-a-codespace/forwarding-ports-in-your-codespace)**

Outra recomendação é gerar um GitHub token, e adicioná-la no cabeçalho da requisição...

Boa sorte! 😉 Se ainda não funcionar, é possível testar outras teorias se você passar mais detalhes sobre a aplicação

**Comment by NayaraNicacio** [comment]
Boa Tarde Leonardo! Eu estou utilizando a url gerada pelo codespace, muito
obrigado pelo retorno me ajudou bastante, consegui resolver com o tutorial
que vc indicou, percebi que a porta estava como privada então ele tava
exigindo um token de acesso, mudei a visualização para publica e funcionou
direitinho.

Em sex., 13 de set. de 2024 às 14:21, Leonardo Felipe Salgado <
***@***.***> escreveu:

> Boa tarde @NayaraNicacio <https://github.com/NayaraNicacio>
>
> Eu não trabalhei muito usando codespaces... 🤔 mas nunca tive esse
> problema. Como você está tentando acessar a aplicação? Pelo localhost ou
> pela URL gerada pelo codespace?
>
> Se for a URL, acredito que o erro de autorização pode estar relacionado...
> ai recomendo fazer um teste usando o Localhost para acessar a aplicação
>
> Tem um tutorial de como fazer isso em *Forwarding-Ports
> <https://docs.github.com/en/codespaces/developing-in-a-codespace/forwarding-ports-in-your-codespace>*
>
> Outra recomendação é gerar um GitHub token, e adicioná-la no cabeçalho da
> requisição...
>
> Boa sorte! 😉 Se ainda não funcionar, é possível testar outras teorias se
> você passar mais detalhes sobre a aplicação
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/138565#discussioncomment-10640378>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACGXU5QCTR3Q537EDVUI24LZWMNKFAVCNFSM6AAAAABOFIAAACVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRUGAZTOOA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 


*Tel: (75) 98827-5012Sempre que te perguntarem se podes fazer um trabalho,*

*respondas que sim e te * *ponhas em seguida a aprender como se faz.*


*F. Roosev**elt*
